src/status_im2/contexts/syncing/standard_authentication/style.cljs

```diff
@@ -0,0 +1,14 @@
+(ns status-im2.contexts.syncing.standard-authentication.style)
```

Comment:
I notice that you are using this `style` namespace in 2 components. We generally try to use 1 style file per view file and if styles need to be reused it is better to do with a component rather and if not possible it is generally better to just duplicate so we are not putting unnecessary coupling between two indeendeny views and their styles

Reply:
Now, It's being used in 1 component.